javascript/ecmascript-2015/string-upgrades/string-extensions.md

```diff
@@ -29,26 +29,26 @@ standards:
 
 ES6 expands on existing primitives to fix some long time issues and add useful functionality.
 
-In previous versions of EcmaScript if you wanted to use Unicode characters one method was to use *String.fromCharCode*:
```

Comment:
I guess we should write it as "ECMAScript": https://en.wikipedia.org/wiki/ECMAScript

Reply:
done